### PR TITLE
[bitnami/solr] Add support for autogenerated certs

### PR DESCRIPTION
**Pull request by alvneiayu**
**Description of the change**

The autogenerated certs support allow us to generate automatically certificates to test or to verify that the TLS/SSL is working properly.

**Benefits**

Make easy to test SSL scenarios.

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/solr/values.yaml [hunk 1]
```diff
@@ -157,6 +160,39 @@ tls:
...
+      ## lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+      ##
+      limits: {}
+      ##   cpu: 100m
+      ##   memory: 128Mi
+      ##
+      requests: {}
+      ##   cpu: 100m
+      ##   memory: 128Mi
+      ##
```
Comment (migruiz4) [inline bitnami/solr/values.yaml:194]: Please check the references to these values.

They are always referred to as `tls.resources`, but here are defined as `tls.image.resources`.

Comment (alvneiayu) [reply bitnami/solr/values.yaml]: good catch. Thanks @migruiz4

**Review by pablogalegoc — APPROVED** [review]
There are conflicts with the `Chart.yaml` but other than that LGTM!

**Comment by pablogalegoc** [comment]
LGTM!

**Pull request merged**